tests/cmds/test_cases.py

```diff
@@ -0,0 +1,74 @@
+from code42cli.main import cli
+
+
+def test_create_case_calls_create_with_expected_params(runner, cli_state):
+    runner.invoke(
+        cli, ["cases", "create", "TEST_CASE"], obj=cli_state,
+    )
+    cli_state.sdk.cases.create.assert_called_once_with(
+        "TEST_CASE", assignee=None, description=None, findings=None, subject=None
+    )
+
+
+def test_update_case_calls_update_with_expected_params(runner, cli_state):
+    runner.invoke(
+        cli, ["cases", "update", "1", "--name", "TEST_CASE2"], obj=cli_state,
+    )
+    cli_state.sdk.cases.update.assert_called_once_with(
+        1,
+        name="TEST_CASE2",
+        status=None,
+        assignee=None,
+        description=None,
+        findings=None,
+        subject=None,
+    )
+
+
+def test_list_calls_get_all_with_expected_params(runner, cli_state):
+    runner.invoke(
+        cli, ["cases", "list"], obj=cli_state,
+    )
+    assert cli_state.sdk.cases.get_all.call_count == 1
+
+
+def test_show_calls_get_case_with_expected_params(runner, cli_state):
+    runner.invoke(
+        cli, ["cases", "show", "1"], obj=cli_state,
+    )
+    cli_state.sdk.cases.get_case.assert_called_once_with(1)
+
+
+def test_export_calls_export_summary_with_expected_params(runner, cli_state):
```

Comment:
As mentioned in another comment, this test creates a file and that should be mocked